allure-report-face/src/main/webapp/js/testcase/testcase.js

```diff
@@ -70,6 +70,12 @@ angular.module('allure.core.testcase.controllers', ['d3'])
 
         findFailedStep($scope.testcase);
 
+        $scope.hasParametersOfKind = function(testcase, kind) {
```

Comment:
it would be better to fetch proper parameters at once instead of doing it on every rendering.
```
$scope.testcaseArguments = testcase.parameters ? testcase.parameters.filter(function (el) {
  return el.kind === "ARGUMENT";
}) : [];
```